### ISSUE TITLE
[Suggestions] Smaller size floor window & more window "dirt" + make it a ship upgrade?

### ISSUE THREAD

**Issue by Fl4sHNoX** [jverif/lc-shipwindow#5]
Overall the mod is a great addition and I want to thank you for all your time and effort before anything else!

Regarding the suggestions:
I really like the floor window, I just wish it was a bit smaller and had a texture that's a bit more "dirty":

![image](https://github.com/jverif/lc-shipwindow/assets/3759901/87e2f103-e4cc-4c71-8cc7-76ae842b9055)

![image](https://github.com/jverif/lc-shipwindow/assets/3759901/319b4dde-9483-4264-bcc2-a4cfc71cb05e)

- The cylinders "touching" the glass circled in red sort of give the feeling of it being less "sturdy" than if they were colliding with the metal floor directly and not the glass

- The blue rectangle gives a better idea of how smaller the window could be, if so the lights under the ship may also need to be brought "closer" towards the window

- I feel like the glass feels a bit "too clean", almost like it's just a hole in the metal sheet, I believe that giving it a bit more "dirt" could probably improve the look and make it "fit" more with the ship's vanilla aesthetic

- Last but not least this feels like it could be a great addition to the Ship's Upgrades in the store, although I don't know how hard it could be to implement, hence why I wouldn't want you to dedicate too much time and effort on this if you feel like it would be too much of a hassle

Thanks again for taking the time to consider these suggestions and keep up the good work!

**Comment by jverif** [comment]
Hello,

I've made some changes to the floor window as suggested. It no longer sits over the supports. I think this does look much better.
![image](https://github.com/jverif/lc-shipwindow/assets/22532950/8f14ea09-5254-4673-a98a-b31beefbde65)

I do think some grime could be a nice detail to the windows. I'll have to experiment with the material to see if I can get something looking good.

Finally, I am actually planning on making the windows ship upgrades, optionally of course. They are a perfect fit for that kind of system.  It shouldn't be too difficult, but I don't have an ETA on that yet.

Thanks for the suggestions!

**Comment by Fl4sHNoX** [comment]
Thank you very much for taking the suggestions into account! 🙏 

Some last few small nitpicks that I forgot to bring up in the original post:

- Actually one of the reasons I suggested to allow for a smaller window was to avoid having the suits rack to have one of its feet stand on the glass like so because it felt "off" (I might have OCD lol):

![image](https://github.com/jverif/lc-shipwindow/assets/3759901/a1f10c1e-8ced-4358-9ae7-b198c9d1fdf7)

- A more substantial nitpick/reason why I suggested an option to change the window size was related to the following idea that there could be more than just one "big window" to make the structure's integrity feel more sturdy:

![image](https://github.com/jverif/lc-shipwindow/assets/3759901/46f77ac0-bf80-41ae-bc4f-b5926001f7d5)

- The following blue line shows the ship's wall angles and I feel like giving the window a shape that sort of "matches" the one of the wall could make it "fit in" a bit more, the green line showing the kinda shape (also stopping at the first beam that's causing some kind of "visual break"):

![image](https://github.com/jverif/lc-shipwindow/assets/3759901/ad577089-ab89-4ccd-bfbd-b684bb55cd94)

- Then, coming back to the previous point before the change of the window shape, if you just took out the part crossed out in red and allowed for two differently sized windows with the steel floor in between, maybe it could give this "reinforced"/"sturdy" look a bit of variation with the window sizes?

![image](https://github.com/jverif/lc-shipwindow/assets/3759901/a1162344-c5f0-4deb-b146-922972b8f9fe)



I hope what I tried to convey makes sense! 🙏  I wish I could experiment a bit more than just drawing lines on a screenshot to see how the ideas actually look in-game to know if they're even actually aesthetically pleasant/viable or not before potentially bothering you with them but so far that's the best I could find!

Regardless, thanks again very much for taking the time to even read all this and take suggestions in the first place! It's very much appreciated!
